go.mod

```diff
@@ -25,3 +25,53 @@ require (
 	k8s.io/code-generator v0.21.1
 	sigs.k8s.io/controller-tools v0.6.0
 )
+
+require (
```

Comment:
How come we have two different `require` sections now?

Reply:
> Because the number of explicit requirements may be substantially larger in an expanded Go 1.17 go.mod file, the newly-added requirements on indirect dependencies in a go 1.17 module are maintained in a separate require block from the block containing direct dependencies. 

https://golang.org/doc/go1.17#go-command